requirements.txt

```diff
@@ -1,9 +1,9 @@
 # Required for fairlearn
-matplotlib>=3.0.3
+matplotlib>=3.1.3
```

Comment:
Does this work? I have memories of this being one of the issues with Python 3.5 on MacOS

Reply:
You're right, but here's the issue: 3.1.3 is for  >=py3.6, but 3.0.3 doesn't work for py3.8
I could exclude python 3.8 from the requirements-fixed build, because that pins it down to the minimum version in this case, or we can drop python 3.5